### DISCUSSION TITLE
Feedback: Note Trait System

### DISCUSSION THREAD

**Discussion by ScriptAutomate** [dendronhq/dendron#1876] [RFCs/Ideas]
Welcome to the feedback discussion on note [traits](https://wiki.dendron.so/notes/EQoaBI8A0ZcswKQC3UMpO/)! Since this is an experimental prototype at this time, we wanted a place for feedback as the feature is being developed and rolled out in phases.

- [Read the RFC for more details](https://docs.dendron.so/notes/E8ZUvTzJ7cVOyZtqHiIKX/)

**Comment by chmac** [comment]
Will traits also work in the cli?

It seems like a potentially powerful feature that could be game changing to automate more complex schemas. Personally, my biggest draw back is accessing my notes on mobile. I'm thinking about writing a few web frontends to add things like daily journal notes, scratch notes, and so on. Traits could be hugely powerful there if they're part of the cli. Then I could create scripts that trigger actions like this, commit + push, and then pull the newly created notes on mobile to edit.

**Reply by kevinslin** [reply]
there's no reason why they shouldn't, it just needs to be implemented. we want the CLI to be a first class client of dendron
